### ISSUE TITLE
Address a11y issues, round 1

### ISSUE THREAD

**Issue by jeffumbricht** [jeffumbricht/jeffumbricht.com#1]
## Issues from automated tests

Issues found using the [axe DevTools chrome extension](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) by [Deque](https://www.deque.com/).

- [x] Contrast
- [x] Links have discernible text (linked icons for Github, linkedin, email)
- [x] Add landmarks

### Contrast

**Primary Button**
Element has insufficient color contrast of 3.74 (foreground color: #ffffff, background color: #ed4933, font size: 10.4pt (13.8667px), font weight: normal). Expected contrast ratio of 4.5:1

**White on blue**
Element has insufficient color contrast of 2.63 (foreground color: #ffffff, background color: #21b2a6, font size: 17.6pt (23.4px), font weight: bold). Expected contrast ratio of 3:1

**Manual check**
Ensure elements over gradient or image pass contrast, even if image fails to load.

**Comment by jeffumbricht** [comment]
## Issues
### Contrast

**Primary Button**
Element has insufficient color contrast of 3.74 (foreground color: #ffffff, background color: #ed4933, font size: 10.4pt (13.8667px), font weight: normal). Expected contrast ratio of 4.5:1

**White on blue**
Element has insufficient color contrast of 2.63 (foreground color: #ffffff, background color: #21b2a6, font size: 17.6pt (23.4px), font weight: bold). Expected contrast ratio of 3:1

**Manual check**
Ensure elements over gradient or image pass contrast, even if image fails to load.

**Comment by jeffumbricht** [comment]
There are still a few "manual" contrast checks identified by the tool as potential issues when looking at text over the main body background image. These should be fine from opacity adjustment made in 9de7381a1af6749600734db290a066457d96199e.
